aries-site/src/layouts/content/Example/Example.js

```diff
@@ -180,7 +180,10 @@ export const Example = ({
         width={width}
         ref={inlineRef}
       >
-        <ThemeContext.Extend value={scaledTheme || theme}>
+        {/* need to overrid theme background so its not resetting */}
```

Comment:
overrid --> override

Does this make the intent any more clear? As currently written, I got the "what is happening" but still had trouble following the "why we want it".

```suggestion
        {/* prevent theme from overriding the desired background color */}
```